best-practices/README.md

```diff
@@ -224,36 +218,29 @@ CSS
 
 Sass
 ----
-
+* Use SCSS syntax.
+* Use [BEM naming conventions].
```

Comment:
Why is this in brackets?

Reply:
Oh, that should be a link to the BEM docs. Will add that.

Reply:
Oh, it is linked! I used the advanced linking syntax, where it matches the string "[BEM...]" to the definition `[BEM naming conventions]: http://getbem.com/introduction/` below. Was just following the link pattern already established in this document.